client/src/index.html

```diff
@@ -2,11 +2,11 @@
 <html>
     <head>
     	<link href='https://fonts.googleapis.com/css?family=Roboto+Slab:700,400,300|Source+Sans+Pro:700,400,300,400italic' rel='stylesheet' type='text/css'>
-    	<link href="main.css" rel="stylesheet" type="text/css">
+    	<link href="/main.css" rel="stylesheet" type="text/css">
         <title>CodeSniff</title>
     </head>
     <body>
         <div id="container"></div>
-        <script src="dist/bundle.js"></script>
+        <script src="/dist/bundle.js"></script>
```

Comment:
It's created when the "gulp" or "gulp dev" is run. It's in the codesniff/client/dist folder.